### DISCUSSION TITLE
Meeting link and schedule

### DISCUSSION THREAD

**Discussion by rartych** [camaraproject/ConnectivityInsights#32] [General]
The link provided in the repo main page: https://verizon.webex.com/verizon/j.php?MTID=ma58c7e3ff9ce05aaa4dd2d758734ef9c produces information like:

> The meeting has been cancelled or ended! 

Would it be possible to resend the Outlook invitation to the mailing list?

**Comment by maheshc01** [comment]
latest invite series has been sent.
